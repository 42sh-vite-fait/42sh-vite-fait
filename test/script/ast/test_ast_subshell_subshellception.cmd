(a | (b1 || b2 | b3 | b4) ; (c1; (c11 && c12) && d))
