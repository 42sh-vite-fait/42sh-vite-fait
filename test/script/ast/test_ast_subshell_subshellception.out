AST:
SUBSHELL: ()
	TERM: ;
		PIPE_SEQUENCE: |
			SIMPLE_COMMAND: a
			SUBSHELL: ()
				AND_OR: ||
					SIMPLE_COMMAND: b1
					PIPE_SEQUENCE: |
						PIPE_SEQUENCE: |
							SIMPLE_COMMAND: b2
							SIMPLE_COMMAND: b3
						SIMPLE_COMMAND: b4
		SUBSHELL: ()
			TERM: ;
				SIMPLE_COMMAND: c1
				AND_OR: &&
					SUBSHELL: ()
						AND_OR: &&
							SIMPLE_COMMAND: c11
							SIMPLE_COMMAND: c12
					SIMPLE_COMMAND: d
